src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/CourtsController.kt

```diff
@@ -96,5 +108,65 @@ class CourtsController(private val courtsService: CourtsService) {
   )
   @GetMapping(value = ["/user-preferences/{username}"], produces = [MediaType.APPLICATION_JSON_VALUE])
   @PreAuthorize("hasAnyRole('BOOK_A_VIDEO_LINK_ADMIN')")
-  fun userPreferences(@PathVariable("username") username: String) = courtsService.getUserCourtPreferences(username)
+  fun getUserCourtPreferences(@PathVariable("username") username: String) = courtsService.getUserCourtPreferences(username)
```

Comment:
I'd have thought we would use the username in the token for this?

Same applies to probation prefs.

Reply:
Yes - changing it now.

Reply:
That's done.